README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+In this scenario, the source of the action is coming from the client input side in the MVC model.  In order to change the color of the song title, some sort of link needs to be placed in the view html.erb file.  Since the controller bridges the gap between the user/router and the views, a controller method makes the most sense.
 ```
```

Comment:
A helper method makes most sense here - they are useful for containing logic that influences what appears in a view.